service/management-ui/src/controllers/course/edit.ts

```diff
@@ -0,0 +1,230 @@
+import * as aws from 'aws-sdk'
+import * as config from 'lib/config'
+import {Request, Response, NextFunction} from 'express'
+import * as request from 'request-promise'
+import * as streamifier from 'streamifier'
+import * as unzip from 'unzip'
+import * as catalog from 'lib/service/catalog'
+import * as elko from 'lib/service/elko'
+import * as log4js from 'log4js'
+import * as template from 'lib/ui/template'
+import {parseString} from 'xml2js'
```

Comment:
Prefer importing the whole module instead of explicit members, if possible — makes it easier to read/understand code later.